MinimedKit/GlucosePage.swift

```diff
@@ -69,8 +68,10 @@ public class GlucosePage {
                 let event = matchEvent(tempOffset, relativeTimestamp: DateComponents())
                 if event is RelativeTimestampedGlucoseEvent {
                     relativeEventCount += 1
-                } else if let event = event as? ReferenceTimestampedGlucoseEvent {
-                    let offsetDate = calendar.date(byAdding: Calendar.Component.minute, value: 5 * relativeEventCount, to: event.timestamp.date!)!
+                } else if let sensorTimestampEvent = event as? SensorTimestampGlucoseEvent,
+                    relativeEventCount == 0 || (event as! SensorTimestampGlucoseEvent).isForwardOffsetReference() {
```

Comment:
I thought I might be able to do that but Xcode was throwing a fit about it. Must have been a stale compiler error because it worked this time around. 👍